docs/CodeReviewRulesOfThumb.md

```diff
@@ -0,0 +1,76 @@
+# Rules of Thumb for Code Reviews
+
+This document aims to outline a set of best practices that are followed, within reason, when reviewing PRs.
+
+## Select a reviewer
+
+Author:
+- Manually assign at least one reviewer when creating a PR
```

Comment:
Does this apply to external contributors?

Reply:
Thanks for pointing this out - didn't think of that. I considered both audiences (the FHIR team and external contributors) in d690eb1.